### DISCUSSION TITLE
cant reboot via web interface

### DISCUSSION THREAD

**Discussion by vincom** [gtxaspec/wz_mini_hacks#258] [Q&A] [answered]
wont  reboot via web interface after making editing some values, after hitting update the  page reloads w/a reboot or revert option, hit reboot and new page loads that states it will refresh in 90sec, page reloads with the said changes made but camera did not reboot and w/out a reboot the new changes dont load, used chrome, firefox, vivaldi but all the same, no reboot

**Comment by virmaior** [comment, marked as answer]
this is my fault. Will fix.

somehow deleted the line that reboots. 
in config.cgi , add `reboot` as a line by itself after line 18.

**Reply by vincom** [reply]
> this is my fault. Will fix.
> 
> somehow deleted the line that reboots. in config.cgi , add `reboot` as a line by itself after line 18.

yup, fixed, no more power cycling or using putty/winscp to making the changes stick, thx

**Comment by Bob-AOL** [comment]
Any explanation for #249?

**Comment by guarddog13** [comment]
I also noticed on the web server that the links listed are both like...

Stream RTSP_HI_RES URL: rtsp://user:pass@192.168.1.31:8554/video2_unicast
Stream RTSP_LOW_RES URL: rtsp://user:pass@192.168.1.31:8554/video2_unicast

**Reply by virmaior** [reply]
in config.js 
should be:
```
  if ((fdata.get('RTSP_HI_RES_ENABLED') == "true") && (fdata.get('RTSP_LOW_RES_ENABLED') == "true")) {
        if (stype == "RTSP_HI_RES") {  stream = "/video1_unicast"  } else { stream ="/video2_unicast" }
  }
```

previously ` ... if (stype == "RTSP_HI") ...`
